docs/templates/reachview/camera-control.mdx

```diff
@@ -0,0 +1,37 @@
+!!! note ""
+	Camera trigger is not available on Reach RS/RS+ and Reach RS2.
```

Comment:
Why do we need it here? This page is only available on Module/M+/M2 docs, as far as I remember